tests/cli/workflow/balsamic/test_compound_commands.py

```diff
@@ -85,16 +96,18 @@ def test_store(
     assert balsamic_context.housekeeper_api.bundle(case_id)
 
 
-def test_start_available(cli_runner: CliRunner, balsamic_context: CGConfig, caplog, mocker):
+def test_start_available(
+    cli_runner: CliRunner, balsamic_context: CGConfig, caplog, mocker, mock_analysis_flow_cell
+):
     """Test to ensure all parts of compound start-available command are executed given ideal conditions
     Test that start-available picks up eligible cases and does not pick up ineligible ones"""
     caplog.set_level(logging.INFO)
 
     # GIVEN CASE ID of sample where read counts pass threshold
-    case_id_success = "balsamic_case_wgs_paired_enough_reads"
+    case_id_success = "balsamic_case_wgs_single"
```

Comment:
Yes it would, but I feel like making these types of changes to the balsamic tests are is not really worth the time, since they have more general problems than hard-coded strings.